autoware_auto_perception_msgs/msg/PredictedObject.idl

```diff
@@ -5,7 +5,7 @@
 module autoware_auto_perception_msgs {
   module msg {
     struct PredictedObject {
-      uint64 object_id;
+      unique_identifier_msgs::msg::UUID object_id;
```

Comment:
This line contains changes in https://github.com/tier4/autoware_auto_msgs/pull/11.

Reply:
Oops
If #11 is merged first, there will be no problem.